### PR TITLE
added ginger milk tea.

### PR DESCRIPTION
**Pull request by VVNoodle**
added _src/recipe/ginger-milk-tea.json_ and _src/recipe/img/ginger-milk-tea.jpg_

Question: Is there a way to add a source for the recipe?

### CODE REVIEW

File: src/recipes/ginger-milk-tea.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "50 gram sugar",
+    "250 ml water"
+  ],
+  "directions": [
+    "Boil all ginger syrup ingredients on a stove top. Reduce heat and simmer for 5-10 minutes untill all sugar has dissolved.",
+    "Strain to get a clear syrup.",
+    "Steep the teabag in the hot water according to the manufacturer instruction.",
+    "Pour the milk and the ginger syrup. Stir to combine. Serve immediately."
+  ],
+  "image": "margarita.jpg"
```
Comment (alfg) [inline src/recipes/ginger-milk-tea.json]:
```suggestion
  "image": "ginger-milk-tea.jpg"
```

**Comment by alfg** [comment]
> Question: Is there a way to add a source for the recipe?
Not currently, but that is a good idea to add to the JSON format.

**Comment by alfg** [comment]
Thanks for the contribution!

Also, I have a list of feature requests I'll be opening as challenges for Hacktoberfest soon. I will add your idea of recipe sources to the list!

**Comment by VVNoodle** [comment]
That's awesome!

Looking forward to contributing more come Hacktoberfest.

**Comment by alfg** [comment]
https://github.com/alfg/opendrinks/issues/43 :)

**Pull request merged**